resources/lib/lcdproc.py

```diff
@@ -319,10 +319,13 @@ def Connect(self):
         self.m_iBigDigits = 0 # No clock
       elif self.m_iColumns < 17:
         self.m_iBigDigits = 5 # HH:MM
-      elif self.m_iColumns < 20:
-        self.m_iBigDigits = 7 # H:MM:SS on play, HH:MM on clock
       else:
-        self.m_iBigDigits = 8 # HH:MM:SS
+        self.m_iBigDigits = 7 # H:MM:SS on play, HH:MM on clock
```

Comment:
Unrelated, please drop. Besides that, what are you trying to fix here?

Reply:
Sorry, it was my local change to not display 6 digit clock on 20-char line LCD, I prefer HH:MM format.

I think, It should be customizable from settings (preferably in `LCD.xml.defaults`, not necessarily from GUI). I'll open new PR with this update on Leia branch.